tests/configs/config_ERP_CORE.py

```diff
@@ -112,8 +112,9 @@
         'unrelated': '`first_stimulus/target` == "unrelated" and '
                      'first_response == "correct"'
     }
-
     contrasts = [('unrelated', 'related')]
+    cluster_forming_t_threshold = 1.5      # Only for testing!
+    cluster_permutation_p_threshold = 0.1  # Only for testing!
```

Comment:
if you put 0.05 nothing survives?

Reply:
Unfortunately, yes, nothing survives :( after all, we're just using 5 participants. Should we consider upping it to 10 or so?